src/main/kotlin/StringCalculator.kt

```diff
@@ -0,0 +1,81 @@
+class StringCalculator() {
+    fun calculate(input: String?): Int {
```

Comment:
kotlin은 강점을 살려 야예 NotNull한 인자를 받으면 어떨까요?

```suggestion
    fun calculate(input: String): Int {
```

Reply:
앗..!! 힌트에 입력값이 null일 경우 예외를 던지라 해서 nullable 하게 선언했는데...
수정하겠습니다~!